instrumentation/aws-sdk/aws-sdk-2.2/testing/src/main/groovy/io/opentelemetry/instrumentation/awssdk/v2_2/AbstractAws2ClientTest.groovy

```diff
@@ -120,12 +132,92 @@ abstract class AbstractAws2ClientTest extends InstrumentationSpecification {
 
     expect:
     response != null
-    assertDynamoDbRequest(service, operation, path, method, requestId)
+    switch (operation) {
+      case "CreateTable":
+        assertCreateTableRequest(path, method, requestId)
+        break
+      case "Query":
+        assertQueryRequest(path, method, requestId)
+        break
+      default:
+        assertDynamoDbRequest(service, operation, path, method, requestId)
+    }
 
     where:
     [service, operation, method, path, requestId, builder, call] << dynamoDbRequestDataTable(DynamoDbAsyncClient.builder())
   }
 
+  def assertCreateTableRequest(path, method, requestId) {
+    assertTraces(1) {
+      trace(0, 1) {
+        span(0) {
+          name "DynamoDb.CreateTable"
```

Comment:
Does it make sense to use the `<db.operation> <db.name>.<table>` naming convention for DynamoDB?

Reply:
Dunno, out of scope of this PR I guess.